docs/source/science/EarthCARE/EarthCare_access and_visualize.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>Link to our other example for this CCI Biomass, which has screen shots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/527/discussion/#comment-2433983502'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>